congame-doc/congame-reference.scrbl

```diff
@@ -450,20 +466,18 @@ is used as the button’s label.
 
 @defmodule[congame/components/transition-graph]
 
-@defform[#:literals (unquote lambda --> goto)
+@defform[#:literals (unquote lambda --> goto done fail)
          (transition-graph transition-clause ...+)
          #:grammar
-         [(transition-clause (code:line [id transition-entry ...+]))
-
-          (transition-entry (code:line --> transition-target))
+         [(transition-clause (code:line [id transition-entry ...+ maybe-lambda]))
```

Comment:
I would change `maybe-lambda` to `maybe-expr` and the definition for the `maybe-expr` term would then be:

```
            (maybe-expr (code:line)
                        (code:line (lambda () transition-expr ...+))
                        (code:line (lambda name:id () transition-expr ...+))
                        (code:line expr))
```